File: Frontend-Angular/src/app/models/Person.ts
export class Person{
    id?: number;
    name: string;
    surname: string;
    description: string;
    profession: string;
    image: string;

    constructor(name: string, surname: string, description: string, profession: string, image: string) {
        this.name = name;
        this.surname = surname;
        this.description = description;
        this.profession = profession;
        this.image = image;
    }
}

File: Backend-Java/src/main/java/com/portfolio/backend/Entity/Person.java
package com.portfolio.backend.Entity;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import javax.validation.constraints.NotNull;

@Entity
@Getter @Setter
public class Person {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column()
    private Long Id;

    @Column(name = "Name")
    @NotNull
    private String Name;

    @Column(name = "Surname")
    @NotNull
    private String Surname;

    @Column(name = "Description")
    @NotNull
    private String Description;

    @Column(name = "Profession")
    @NotNull
    private String Profession;

    @Column(name = "Image")
    private String Image;

}
